homeassistant/components/mqtt/discovery.py

```diff
@@ -197,9 +199,14 @@ class MQTTConfig(dict):
             config_entries_key = "{}.{}".format(component, "mqtt")
             async with hass.data[DATA_CONFIG_ENTRY_LOCK]:
                 if config_entries_key not in hass.data[CONFIG_ENTRY_IS_SETUP]:
-                    await hass.config_entries.async_forward_entry_setup(
-                        config_entry, component
-                    )
+                    if component == "device_automation":
+                        from . import device_automation
```

Comment:
We should just always import it ?

Reply:
Then we get circular dependencies because device_automation is importing mqtt..

Reply:
Let's add a comment.